php1/mysql/userdb.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$servername = "localhost";
+$username = "root";
+$password = "mynewpassword";
+$dbname = "User";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $dbname);
+// Check connection
+if ($conn->connect_error) {
+  die("Connection failed: " . $conn->connect_error);
+}
+
+// $conn->close();
```

Comment:
Why you comment this can you elaborate please?

Reply:
I am including this connection page wherever I had to write SQL queries so if I close the connection then the query will not get executed that's the reason I have commented on that line.